deps.edn

```diff
@@ -5,7 +5,7 @@
   org.clojure/data.zip {:mvn/version "1.0.0"}
   org.clojure/tools.cli {:mvn/version "1.0.206"}
   org.clojure/tools.deps.alpha {:mvn/version "0.12.1003"}
-  cheshire/cheshire {:mvn/version "5.10.1"}
+  org.clojure/data.json {:mvn/version "2.4.0"}
```

Comment:
If this is meant to be kept alphabetized (within org.clojure), easy change

Reply:
No problem :)